src/main/java/com/github/webicitybrowser/spec/fetch/imp/FetchConsumeBodyActionImpl.java

```diff
@@ -0,0 +1,15 @@
+package com.github.webicitybrowser.spec.fetch.imp;
+
+import com.github.webicitybrowser.spec.fetch.FetchConsumeBodyAction;
+import com.github.webicitybrowser.spec.fetch.FetchResponse;
+import com.github.webicitybrowser.spec.fetch.builder.FetchResponseBuilder;
+import com.github.webicitybrowser.spec.fetch.builder.imp.FetchResponseBuilderImp;
+
+public class FetchConsumeBodyActionImpl implements FetchConsumeBodyAction {
```

Comment:
The interface is meant to be implemented elsewhere. This implementation class can be deleted